conf.py

```diff
@@ -240,8 +240,13 @@
 htmlhelp_basename = 'PythonPatternsdoc'
 
 # -- Options for LaTeX output ---------------------------------------------
+latex_engine = 'xelatex'
```

Comment:
Could you explain a bit about the `xelatex` engine? I have not heard of it before. Thanks!

Reply:
It's a modern typesetting engine for `LaTeX` which supports unicode. There are a lot of unicode characters in this project which are not supported by Sphinx's default `LaTeX` engine (`pdflatex`).

Reply:
Ah, thanks for clarifying! For some reason a search had led me to an indexing tool for LaTeX, but I probably mis-navigated Ubuntu's packaging search command line tool.